aux_assistant.py

```diff
@@ -40,8 +41,6 @@
 aux_assistant_playlist = list()
 saved = list()
 global_playlist_names = set()
-USER = None
```

Comment:
Good that we are no longer using the global variable with flask login being implemented.